debugging-via-dotnet-dump/test.sh

```diff
@@ -89,6 +89,8 @@ heading "Installing dotnet-dump"
 
 dotnet tool uninstall -g dotnet-dump || true
 dotnet tool install -g dotnet-dump
+# For preview releases:
+#dotnet tool install -g dotnet-dump --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json
```

Comment:
Maybe this can be done if an ENV var is set? Something so that it can b enabled/disabled without having to change the code.

Reply:
Technically, this addition is not even necessary: the test runner injects this source (where preview versions are available) when running this test anyway.

It's only when you are debugging this script itself (without using the test runner) that this additional source plays a part. And at that point, you are probably changing the code too. Reading/enabling the env var about be about as much effort, it seems to me.

 Would you be okay with leaving it as is? Perhaps I should remove this completely?